test/controllers/partials/BtaHomeControllerSpec.scala

```diff
@@ -97,4 +97,64 @@ class BtaHomeControllerSpec extends ControllerBaseSpec {
       }
     }
   }
+
+  "Calling the .claimEnrolment action" when {
+
+    "A user is logged in and enrolled to HMCE-VATDEC-ORG" should {
+
+      val goodEnrolments: Enrolments = Enrolments(
+        Set(
+          Enrolment(
+            "HMCE-VATDEC-ORG",
+            Seq(EnrolmentIdentifier("VRN", "123456789")),
+            "Active")
+        )
+      )
+
+      "return 200" in new Test {
+        override val authResult: Future[Enrolments] = Future.successful(goodEnrolments)
+        private val result = target.vatSection()(fakeRequest)
+        status(result) shouldBe Status.OK
+      }
+
+      "return HTML" in new Test {
+        override val authResult: Future[Enrolments] = Future.successful(goodEnrolments)
+        private val result = target.vatSection()(fakeRequest)
```

Comment:
All of these additional tests are calling the `vatSection()` function and not the `claimEnrolment()` function.